bin/sl-deploy.txt

```diff
@@ -6,8 +6,11 @@ Options:
   -h,--help          Print this message and exit.
   -v,--version       Print version and exit.
   -s,--service SVC   Deploy to service SVC.
+  -z,--size SIZE     Set size of cluster to SIZE before deploy.
 
 Arguments:
+  SIZE: Positive integer value.  Defaults to the number of cpus.
```

Comment:
doesn't have to be a number, also, doesn't have a default (default is to not change the size)